<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMkb10Widget</class>
 <widget class="QWidget" name="MainMkb10Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Поиск по описанию</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_filter"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_search">
       <property name="text">
        <string>Искать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_clear">
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="m_stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="m_treeWidget">
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListView" name="m_listView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
